tools/db/move-team/README.md

```diff
@@ -0,0 +1,17 @@
+# move-team - export/import teams
+
+The executable `move-team` can..
+- ... export all data that belongs to a team to a file dump
+- ... import files json files to cassandra
+- ... export all data from some tables. This was only intended for development.
+
+The script `debug_merge_teams.sh` can be used to replace all team ids in a file
+with a a single team id. This can be useful for testing.
+
+`src/Schema.hs` is created by the executable `gen-table-types`.
```

Comment:
add gen-table-types to `stack.yaml`?  rename to eg. `move-team-generate`?

Reply:
Renamed to `move-team-generate`. `stack exec move-team-generate` works